lib/src/tabs/tab-set.component.scss

```diff
@@ -6,16 +6,15 @@
 }
 
 .vertical-container {
-    display: table;
     height: 100%;
     width: 100%;
+    display: flex;
```

Comment:
I have a feeling we're going to go back and forth on this until the end of time. ;)